cookbook/Chapter 4 - Find out on which weekday people bike the most.py

```diff
@@ -1,97 +1,65 @@
-# %%
-import pandas as pd
+import polars as pl
 import matplotlib.pyplot as plt
 
-# Make the graphs a bit prettier, and bigger
+# Set matplotlib styles
 plt.style.use("ggplot")
 plt.rcParams["figure.figsize"] = (15, 5)
 plt.rcParams["font.family"] = "sans-serif"
 
-# This is necessary to show lots of columns in pandas 0.12.
-# Not necessary in pandas 0.13.
-pd.set_option("display.width", 5000)
-pd.set_option("display.max_columns", 60)
-
 # %% Load the data
-bikes = pd.read_csv(
+bikes = pl.read_csv(
     "../data/bikes.csv",
     sep=";",
```

Comment:
Hey Joe, i think in polars should be separator=";"

Reply:
Thanks, fixed! :)